installers/MacInstaller.hs

```diff
@@ -35,10 +42,29 @@ main = do
   copyFile "cardano-node" (dir <> "/cardano-node")
   copyFile "log-config-prod.yaml" (dir <> "/log-config-prod.yaml")
   copyFile "data/ip-dht-mappings" (dir <> "/ip-dht-mappings")
+  copyFile "data/ip-dht-mappings" (dir <> "/ip-dht-mappings")
+  copyFile "build-certificates-unix.sh" (dir <> "/build-certificates-unix.sh")
+  copyFile "tls/ca.conf" (dir <> "/ca.conf")
+  copyFile "tls/server.conf" (dir <> "/server.conf")
 
   -- Rewrite libs paths and bundle them
   _ <- chain dir $ fmap T.pack [dir <> "/cardano-launcher", dir <> "/cardano-node"]
 
+  -- Prepare postinstall script
+  createDirectoryIfMissing False resDir
```

Comment:
You're right!  There was a log of change jitter, and this is remains of it.

Thank you for spotting itQ